### PR TITLE
create category

### PR DESCRIPTION
**Pull request by thucpv-1748**
https://edu-redmine.sun-asterisk.vn/issues/10414

### CODE REVIEW

File: resources/assets/css/frontend/gozha-nav.css [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+            transform: scaleY(1);
+        }
+    }
+}
+
+@media (max-width: 992px) {
+
+    /*javascript use it*/
+    ul#navigation.hidden-c{
+        display:none;
```
Comment (BunnyPi04) [inline resources/assets/css/frontend/gozha-nav.css]: convention

File: resources/assets/css/frontend/style.css [hunk 2]
```diff
@@ -0,0 +1,9979 @@
...
+  font-size: 75%;
+  line-height: 0;
+  position: relative;
+  vertical-align: baseline;
+}
+sup {
+  top: -0.5em;
+}
+sub {
+  bottom: -0.25em;
```
Comment (BunnyPi04) [inline resources/assets/css/frontend/style.css:78]: convention

File: resources/assets/css/frontend/touch.css [hunk 3]
```diff
@@ -0,0 +1,37 @@
+@media (max-width: 640px){
```
Comment (BunnyPi04) [inline resources/assets/css/frontend/touch.css]: convention

**Review by BachFl2w — APPROVED** [review]

File: resources/views/layout/frontend/category.blade.php [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+@endsection
+
+@section('javascript')
+    <!-- jQuery UI -->
+    <link href="{{ URL::asset('jquery-ui/external/jquery-1.10.2/jquery.js') }}" rel="stylesheet"/>
+    <!-- Magnific-popup -->
+    <script src="{{ URL::asset('js/frontend/external/jquery.magnific-popup.min.js') }}"></script>
+
+    <!--*** Google map  ***-->
+    <script src="https://maps.google.com/maps/api/js?sensor=true"></script>
```
Comment (chinn-2017) [inline resources/views/layout/frontend/category.blade.php:118]: :v cái gì đây :v link online ak =)))

Comment (thucpv-1748) [reply resources/views/layout/frontend/category.blade.php:118]: google map ma????

**Pull request merged**